src/blockchain/deposit_strategy/deposit_transaction_sender.py

```diff
@@ -28,50 +42,115 @@ def _prepare_signs_for_deposit(quorum: list[DepositMessage]) -> tuple[tuple[str,
 
     def prepare_and_send(
         self,
+        module_id: int,
```

Comment:
Let's use order params like in `prepare_and_send`
Or also change `prepare_and_send` annotation to (int, list)

Reply:
Done